include/package-ipkg.mk

```
@@ -218,7 +219,14 @@ $(_endef)
 	$(if $(PROVIDES),@for pkg in $(filter-out $(1),$(PROVIDES)); do cp $(PKG_INFO_DIR)/$(1).provides $(PKG_INFO_DIR)/$$$$pkg.provides; done)
 	$(CheckDependencies)
 
+    ifneq ($$(CONFIG_EXTRACT_DEBUGINFO),)
+	[ ! -d $$(DBGINFO_$(1)) ] || rm -rf $$(DBGINFO_$(1))
+	mkdir $$(DBGINFO_$(1))
```

```suggestion
	mkdir -p $$(DBGINFO_$(1))
```

Failed for me without compiling a lib.

---

yes - good point!
i always have bin/ populated quite densely and never noticed.